test/server.test.js

```diff
@@ -3,6 +3,15 @@
 process.env.beacon_hash ||= 'HASH_FROM_TESTING'
 
 const { test } = require('tap')
+
+// manipulate time like nano-cache's test do, see here
+// https://github.com/akhoury/nano-cache/blob/8f7f9fe1c2883ac092cf768e1df4e3d62e597648/test/test.js#L5-L9
+const NanoCache = require('nano-cache')
+let fakeNow = null;
+NanoCache.prototype.now = function () {
+    return fakeNow || (new Date()).getTime();
+};
```

Comment:
I tried using [@sinonjs/fake-timers](https://www.npmjs.com/package/@sinonjs/fake-timers) to simulate the passage of time, but couldn't get it working, so I went digging to see how `nano-cache` itself tests its TTL, and adapted that here.